src/anomalib/models/cflow/lightning_model.py

```diff
@@ -173,6 +175,10 @@ def validation_step(self, batch: dict[str, str | Tensor], *args, **kwargs) -> ST
         batch["anomaly_maps"] = self.model(batch["image"])
         return batch
 
+    @property
+    def trainer_arguments(self) -> dict[str, Any]:
+        return {"gradient_clip_val": 0, "max_epochs": 1, "num_sanity_val_steps": 0}
```

Comment:
Is this correct? 
https://github.com/openvinotoolkit/anomalib/blob/c5d333355a6870f3a2e79db95b449876bead75bb/src/anomalib/models/cflow/config.yaml#L87C3-L88C1